### PR TITLE
JCA test cases

### PR DESCRIPTION
**Pull request by cypo721**
Updated BasePage methods.
Created jca increment/decrement policy management testcases.

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/page/config/JCAPage.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/JCAPage.java]: Please remove unused import.

File: common/src/main/java/org/jboss/hal/testsuite/page/config/JCAPage.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * Created by pcyprian on 24.8.15.
+ */
+public class JCAPage extends ConfigurationPage {
+
+    @Drone
+    private WebDriver browser;
+
+    private static final By CONTENT = By.className(PropUtils.get("page.content.rhs.class"));
+
+    public ResourceAdaptersFragment getContent(){
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/JCAPage.java]: This is not used. Do you plan to use this method in further development?

File: common/src/main/java/org/jboss/hal/testsuite/page/BasePage.java [hunk 3]
```diff
@@ -279,6 +280,23 @@ public BasePage selectMenu(String label) {
         return this;
     }
 
+    public void switchToEditMode(){
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/BasePage.java]: Is this method (and #waitUntilPanelIsVisible bellow) needed any more?

**Comment by zebrik** [comment]
I personally don't like dependent tests (and failures) but we are in a hurry so I'm ok with that for now. So apart of some minor inline questions this LGTM.

**Comment by cypo721** [comment]
Unused methods removed and made checkstyle fixes.

**Comment by zebrik** [comment]
It seems that checkstyle fixes haven't been pushed yet. 
DatasourceXAPolicyTestCase.java:39:41: '=' is not followed by whitespace.
DatasourceXAPolicyTestCase.java:56:31: '{' is not preceded with whitespace.
DatasourceXAPolicyTestCase.java:91:38: '{' is not preceded with whitespace.
etc.

**Comment by cypo721** [comment]
Fixed chceckstyle problems. Commit squashed.

**Pull request merged**